src/core/shape_inference/include/squeeze_shape_inference.hpp

```diff
@@ -103,6 +117,43 @@ std::vector<TRShape> shape_infer(const Squeeze* op,
     } else {
         output_shape = PartialShape::dynamic();
     }
+}
+}  // namespace util
+namespace v0 {
+/**
+ * \brief Do Squeeze shape inference.
```

Comment:
```suggestion
 * @brief Do Squeeze shape inference.
```
Detail use `@` instead `\` is more common in OV for this doxy style.

Reply:
Done